src/main/java/com/iota/iri/controllers/AddressViewModel.java

```diff
@@ -51,12 +51,9 @@ public static AddressViewModel load(Tangle tangle, Indexable hash) throws Except
     }
 
     /**
-     * Stores the {@link Address} set indexed by its {@link Hash} identifier in the database.
-     *
-     * @param tangle The tangle reference for the database
-     * @return True if the object was saved correctly, False if not
-     * @throws Exception Thrown if the {@link Address} set or index {@link Hash} are null
+     * {@inheritDoc}
```

Comment:
if you don't want to add anything to the doc, then you don't even need this annotation. it will inherit the docs automatically. `@inheritDoc` is used when you want to add additional information.

Reply:
this is relevant for this whole PR.